cupy/fft/_fft.py

```diff
@@ -1102,6 +1100,4 @@ def ifftshift(x, axes=None):
         axes = list(range(x.ndim))
     elif isinstance(axes, np.compat.integer_types):
         axes = (axes,)
-    for axis in axes:
-        x = cupy.roll(x, -(x.shape[axis] // 2), axis)
-    return x
+    return cupy.roll(x, [-x.shape[axis] // 2 for axis in axes], axes)
```

Comment:
Thanks for your contribution! Seems that our CI fails in ifftshift and we'd need these parentheses so we have appropriate rounding:

```suggestion
    return cupy.roll(x, [-(x.shape[axis] // 2) for axis in axes], axes)
```

Reply:
Thanks, you're quite right. I had only properly tested this with fftshift, since I always use even sized (and in fact power of two sized) arrays in my own code. I missed the difference that the parentheses would make.